crates/ruff_linter/src/rules/ruff/rules/missing_fstring_syntax.rs

```diff
@@ -70,13 +70,38 @@ pub(crate) fn missing_fstring_syntax(
         }
     }
 
+    // We also want to avoid expressions that are intended to be translated.
+    if semantic.current_expressions().any(is_gettext) {
```

Comment:
I had to generalize this to look at all parent expressions, to catch cases like: `print(_("formatting of {partial} in a translation string is bad practice"))`

Reply:
Great additions! Thanks.